Toolchain/ResignQemuSystemForBigSur.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -eu
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+BUILD="$DIR/../Build"
+ENTITLEMENT_FILE="$BUILD/entitlements.xml"
+
+SYSTEM_NAME="$(uname -s)"
+QEMU_SYSTEM_X86_64="$(command -v qemu-system-x86_64)"
+
+if [ "$SYSTEM_NAME" = "Darwin" ]; then
```

Comment:
Two comments on this:

I'm assuming this applies to macOS versions after Big Sur, like Monterey as well? So the script name and docs might need a tweak to reflect that. E.g. `EntitleQemuToMacOSHVF.sh`. and "if you're running macOS Big Sur or newer..."

Second, is there a way to check if qemu already has this entitlement to not blast away some other entitlements it might have if e.g. it's shipped by homebrew instead of built from source?

Reply:
Looks like this is already the default as of qemu 6.0.0: https://github.com/qemu/qemu/commit/8a74ce618b647e71ac703b5fe5dba3a2329a4d76. It's just the documentation was out-of-date.

Reply:
Aha that's even better. Could you squash+fixup your updated commit into the first one per the contribution guidelines though? Then the PR would only have the one commit, that removes the section from the docs.

Reply:
Updated. Thanks for the review.